.github/workflows/e2e.yml

```diff
@@ -12,8 +12,8 @@ on:
       - develop
 
 env:
-  GreenfieldTag: v1.1.0
-  GreenfieldStorageProviderTag: v1.0.5
+  GreenfieldTag: develop
```

Comment:
why change it to develop?

Reply:
greenfield chain add "tags" in the object/bucket/group types in its develop branch. We can change it after it releases a formal version. Otherwise the e2e will fail